README.md

```diff
@@ -1,9 +1,11 @@
 Ember.js
 ========
 
-Shim repository for [Ember Application Framework](http://emberjs.com/).
+_Shim repository for [Ember Application Framework](http://emberjs.com/)._
 
 This package provides the core of the ember.js framework.
+When a new [build is released](http://emberjs.com/blog/2013/09/06/new-ember-release-process.html),
+package managers will find it here.
```

Comment:
I am not sure how to word it, but we should make it clear that this repo is for usage with package managers that do not have "their own storage".  Specifically speaking, we do not plan to use this repo when we publish and consume Ember from NPM (#soon).

Can you think of a good way to say that?